src/code/code_800E4FE0.c

```diff
@@ -442,12 +464,12 @@ void Audio_ProcessCmd(AudioCmd* cmd) {
             return;
         }
         if (cmd->arg1 == 0xFF) {
-            phi_v0 = gAudioContext.unk_5BDC[cmd->arg0];
-            for (i = 0; i < 0x10; i++) {
-                if (phi_v0 & 1) {
+            activeChannelsFlags = gAudioContext.activeChannelsFlags[cmd->arg0];
+            for (i = 0; i < 16; i++) {
+                if (activeChannelsFlags & 1) {
```

Comment:
I think `activeChannelsFlags` is too broad, it's just "on which channels to run `CHAN_UPD_` cmds"

Reply:
Right, will revert this until these commands are better documented then